### ISSUE TITLE
harbor.yml log-level warning breaks registry log-level assumptions

### ISSUE THREAD

**Issue by Mierpo** [goharbor/harbor#8192]
**Expected behavior**
Expected behavior:
Log level "warning" provided to harbor.yml configures all containers correctly.

**Current behavior**
Providing this in harbor.yml:

```
log:
  level: warning
```
Causes registry.log to be filled with:

> configuration error: error parsing /etc/registry/config.yml: Invalid loglevel warning Must be one of [error, warn, info, debug]

And the container keeps restarting:

```
CONTAINER ID        IMAGE                                               COMMAND                  CREATED             STATUS                          PORTS                                      NAMES
e12f163ga7b5        goharbor/registry-photon:v2.7.1-patch-2819-v1.8.1   "/entrypoint.sh /etc…"   18 minutes ago      Restarting (1) 39 seconds ago                                              registry
```

On the other hand, providing this in harbor.yml:

```
log:
  level: warn
```

Causes installation to fail (as expected, when running install.sh)
> Traceback (most recent call last):
  File "main.py", line 62, in <module>
    main()
  File "/usr/lib/python3.6/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3.6/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3.6/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "main.py", line 31, in main
    config_dict = parse_yaml_config(conf)
  File "/usr/src/app/utils/configs.py", line 191, in parse_yaml_config
    raise Exception('log level must be one of debug, info, warning, error, fatal')
Exception: log level must be one of debug, info, warning, error, fatal

**Steps to reproduce the problem:**
Just follow the instructions for installing harbor using the offline installer for version 1.8.1, and set your log level to warning in harbor.yml.
Offline installer can be found here:
https://storage.googleapis.com/harbor-releases/release-1.8.0/harbor-offline-installer-v1.8.1.tgz

**Versions:**
Please specify the versions of following systems.
- Harbor version: 1.8.1 (offline installer)
- Docker version 18.09.7
- docker-compose version 1.24.0

**Quick fix**
Don't use warning/warn as your log level. We changed to info level and everything works as expected!

**Comment by d2dyno** [comment]
Just found same issue here.

**Comment by ninjadq** [comment]
fixed in #8590
